erpnext/accounts/report/trial_balance/trial_balance.html

```
@@ -1 +1,215 @@
-{% include "accounts/report/financial_statements.html" %}
+{%
+	const report_columns = report
+		.get_columns_for_print()
+		.filter(col => !col.hidden);
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)